services/api/src/resources/user/sql.ts

```diff
@@ -14,4 +14,10 @@ export default {
       .andWhere('sk.key_type', keyType)
       .select('user_ssh_key.usid')
       .toString(),
+  deleteUserSshKeys: (id: string) =>
```

Comment:
I think this leaves behind records in the `user_ssh_key` table?

Reply:
ah yep